pkg/webui/components/header-v2/story.js

```diff
@@ -0,0 +1,63 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react'
+
+import TtsLogo from '@assets/static/logo.svg'
```

Comment:
Let's update this one with the new logo design from @pierrephz as well. You can find it in the design repo.

Reply:
I think I got confused here. The TTS logo is not used in the header anymore anyway, so it can just be removed as prop.